### PR TITLE
NXP-30963: disable blob buttons, add layouts and enable document page

### PR DESCRIPTION
**Pull request by richardsd**

### CODE REVIEW

File: nuxeo-compound-documents-web/nuxeo-compound-documents.html [hunk 1]
```diff
@@ -14,3 +14,44 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+<!-- Disable blob buttons -->
+<nuxeo-slot-content name="replaceBlobAction" slot="BLOB_ACTIONS" order="35" disabled>
```
Comment (Gabez0r) [inline nuxeo-compound-documents-web/nuxeo-compound-documents.html]: not needed
```suggestion
<nuxeo-slot-content name="replaceBlobAction" slot="BLOB_ACTIONS" disabled>
```

File: nuxeo-compound-documents-web/nuxeo-compound-documents.html [hunk 2]
```diff
@@ -14,3 +14,44 @@
...
+  <template>
+    <nuxeo-replace-blob-button document="[[document]]" xpath="[[xpath]]"></nuxeo-replace-blob-button>
+  </template>
```
Comment (Gabez0r) [inline nuxeo-compound-documents-web/nuxeo-compound-documents.html]: This is not needed
```suggestion
```

File: nuxeo-compound-documents-web/nuxeo-compound-documents.html [hunk 3]
```diff
@@ -14,3 +14,44 @@
...
+</nuxeo-slot-content>
+
+<nuxeo-slot-content name="deleteBlobAction" slot="BLOB_ACTIONS" order="40" disabled>
```
Comment (Gabez0r) [inline nuxeo-compound-documents-web/nuxeo-compound-documents.html]: This is not needed
```suggestion
<nuxeo-slot-content name="deleteBlobAction" slot="BLOB_ACTIONS" disabled>
```

File: nuxeo-compound-documents-web/nuxeo-compound-documents.html [hunk 4]
```diff
@@ -14,3 +14,44 @@
...
+  <template>
+    <nuxeo-delete-blob-button document="[[document]]" xpath="[[xpath]]"></nuxeo-delete-blob-button>
+  </template>
```
Comment (Gabez0r) [inline nuxeo-compound-documents-web/nuxeo-compound-documents.html]: This is not needed
```suggestion
```

**Review by Gabez0r — COMMENTED** [review]
looking good 💪

File: nuxeo-compound-documents-web/nuxeo-compound-documents.html (hunk 2 above)
Comment (richardsd) [reply nuxeo-compound-documents-web/nuxeo-compound-documents.html]: Ah, you read my mind. I was going to ask if I should keep the actual content of the contribution just for reference, or I should remove it. Thanks

File: nuxeo-compound-documents-web/nuxeo-compound-documents.html [hunk 5]
```diff
@@ -14,3 +14,36 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+<!-- Disable blob buttons -->
+<nuxeo-slot-content name="replaceBlobAction" slot="BLOB_ACTIONS" disabled></nuxeo-slot-content>
+
+<nuxeo-slot-content name="deleteBlobAction" slot="BLOB_ACTIONS" disabled></nuxeo-slot-content>
+
+<!--  Display document page for CompoundDocument facet -->
+<nuxeo-slot-content name="documentViewPage" slot="DOCUMENT_VIEWS_PAGES" order="10" priority="1">
```
Comment (Gabez0r) [inline nuxeo-compound-documents-web/nuxeo-compound-documents.html]: Not sure we even need the bumped priority here?

File: nuxeo-compound-documents-web/nuxeo-compound-documents.html [hunk 6]
```diff
@@ -14,3 +14,36 @@
...
+    >
+      <template>
+        <nuxeo-document-page name="view" document="[[document]]" opened></nuxeo-document-page>
+      </template>
+    </nuxeo-filter>
+  </template>
+</nuxeo-slot-content>
+
+<!-- Prevent collapsible document page from being displayed in documents with the CompoundDocument facet -->
+<nuxeo-slot-content name="documentCollapsiblePage" slot="DOCUMENT_VIEWS_PAGES" order="10" priority="1">
```
Comment (Gabez0r) [inline nuxeo-compound-documents-web/nuxeo-compound-documents.html]: Same here

File: nuxeo-compound-documents-web/nuxeo-compound-documents.html (hunk 5 above)
Comment (richardsd) [reply nuxeo-compound-documents-web/nuxeo-compound-documents.html]: I will remove the priority.

File: nuxeo-compound-documents-web/nuxeo-compound-documents.html (hunk 6 above)
Comment (richardsd) [reply nuxeo-compound-documents-web/nuxeo-compound-documents.html]: I will remove the priority.

**Review by Gabez0r — APPROVED** [review]

**Pull request merged**